miden-lib/asm/kernels/transaction/lib/account.masm

```diff
@@ -402,6 +404,64 @@ export.validate_storage_offsets
     # => []
 end
 
+#! Validates all account procedures sizes by checking that:
+#! - All storage sizes are in bounds
+#! - All procedures not accessing storage have offset and size set to 0
+#!
+#! Stack: []
+#! Output: []
+export.validate_storage_sizes
```

Comment:
Why have a separate procedure for this rather than incorporate the logic into `validate_storage_offsets`? Unless it causes some complications, looping over procedure infos only once should be more efficient.